src/DetailsView/reports/components/automated-check-issue-details-card.scss

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+@import '../../../common/styles/colors.scss';
+@import '../../../common/styles/fonts.scss';
+
+.base-card-layer-main {
+    position: relative;
+    opacity: 1;
+    background: #fff;
```

Comment:
can we use a variable for the color here?

Reply:
Sure. Forgot to  put in variabl.e